index.js

```diff
@@ -0,0 +1,5 @@
+if (process.env.NODE_ENV === 'production') {
```

Comment:
For npm consumers, the `process.env.NODE_ENV` would be something that's set by their bundlers.  I think most bundlers do this by default these days, but might be worth calling out in a "building for production section" in the docs?

Reply:
I'm actually following [the same pattern as React](https://unpkg.com/react@16.4.1/index.js) here. I think it's really smart that they export the production or development version of the package depending on the `NODE_ENV` of the build itself, which – you're right – is set by the bundler. We could serve just the development version on its own, and expect our users to properly minify the code for production, or we can safeguard against it in a non-harmful way

Reply:
I think the pattern is good, I was just thinking that it might be helpful to at least include a note in the readme that we're expecting users to set this up if they want our production build.  React has a similar [section](https://reactjs.org/docs/optimizing-performance.html#brunch) on this in their docs.

I also feel like it could be reasonable to rely on users to set up their own minification step in their build process.  I think a lot of libraries go this route.  I think React does some pretty aggressive optimizations that may not be fully applicable to us.  

I'm fine with either!

Reply:
Ah I see what you're saying. Hmmm, good point… What would you do?

Reply:
Hm, personally I think I would just point to non-minified code.